client/my-sites/earn/memberships/products.jsx

```diff
@@ -392,11 +405,14 @@ class MembershipsProductsSection extends Component {
 							'Add a custom message to the confirmation email that is sent out for this recurring payment plan.'
 						) }
 					</p>
-					<FormTextArea
+					<CountedTextarea
 						value={ editedCustomConfirmationMessage }
 						onChange={ ( event ) =>
 							this.setState( { editedCustomConfirmationMessage: event.target.value } )
 						}
+						acceptableLength={ MAX_LENGTH_CUSTOM_CONFIRMATION_EMAIL_MESSAGE }
+						showRemainingCharacters={ true }
+						placeholder={ translate( 'Write a short welcome message' ) }
```

Comment:
"Write a short welcome message. This will be shared in the confirmation email sent to subsriber after successful checkout."

Reply:
Wouldn't the description placed before the textarea field be a better fit for that extra copy?

<img width="801" alt="Screen Shot 2020-05-22 at 13 58 23" src="https://user-images.githubusercontent.com/1233880/82679722-9b49b280-9c4b-11ea-8cb1-b1ea22c63849.png">

It's actually quite similar to the current one, so we might just want to rephrase the existing one.

The placeholder was intended for encouraging users to write a short message to avoid lengthy emails and any possibility of email clipping.

Reply:
I would make the placeholder actually placeholder text and let the label/description speak to the purpose of the textarea.

And there's already quite a bit of copy on this screen so if we can keep it as short as possible while still communicating everything we want to say.

> **Description:** "Add a short message to the confirmation email sent to subscribers."
> **Placeholder:** "Thank you for subscribing!"

Thoughts?

This isn't blocking for me and we can update all the copy in this modal in a follow-up.

Reply:
@danhauk I like that, added in 5217378d59467d25d3476d4255d52686f088f143. I'll merge, and we can re-evaluate all copy again as followup anyways.

Reply:
Re placeholder copy that sounds reasonable to me